Test/triggers/loop-detection-messages--unit-tests.dfy.expect

```diff
@@ -1,28 +1,28 @@
 loop-detection-messages--unit-tests.dfy(11,9): Info: Selected triggers: {f(f(i))}
- Rejected triggers: {f(i)} (may loop with "f(f(i))") Trigger position: loop-detection-messages--unit-tests.dfy(11,36)
-loop-detection-messages--unit-tests.dfy(12,9): Info: Selected triggers: {f(_t#0), f(i)} Trigger position: loop-detection-messages--unit-tests.dfy(12,36)
-loop-detection-messages--unit-tests.dfy(13,9): Info: Selected triggers: {f(_t#0), f(i)} Trigger position: loop-detection-messages--unit-tests.dfy(13,52)
+ Rejected triggers: {f(i)} (may loop with "f(f(i))")
+loop-detection-messages--unit-tests.dfy(12,9): Info: Selected triggers: {f(_t#0), f(i)}
+loop-detection-messages--unit-tests.dfy(13,9): Info: Selected triggers: {f(_t#0), f(i)}
 loop-detection-messages--unit-tests.dfy(15,9): Info: For expression "false ==> f(i) == f(_t#0)":
-   Selected triggers: {f(_t#0), f(i)} Trigger position: loop-detection-messages--unit-tests.dfy(15,36)
+   Selected triggers: {f(_t#0), f(i)} in sub-expression at loop-detection-messages--unit-tests.dfy(15,36)
 loop-detection-messages--unit-tests.dfy(15,9): Info: For expression "false ==> f(i) == g(i)":
    Selected triggers:
-     {g(i)}, {f(i)} Trigger position: loop-detection-messages--unit-tests.dfy(15,54)
+     {g(i)}, {f(i)} in sub-expression at loop-detection-messages--unit-tests.dfy(15,54)
```

Comment:
Same here, two triggers and one position

Reply:
We are reporting the position of the token "==", which is the token held by the binaryExpression "f(i) == g(i)" that cannot be further divided.